src/iedit.cpp

```diff
@@ -213,6 +213,42 @@ void iedit_disp_cybereyes_menu(struct descriptor_data *d)
   send_to_char(CH, "Set Options: ^c%s^n\r\nEnter options (0 to quit): ", buf1);
 }
 
+void iedit_disp_cyberarms_menu(struct descriptor_data *d)
+{
+  CLS(CH);
+  for (int y = 0; y < NUM_ARMSMODS; y += 2)
+    send_to_char(CH, "%2d) %-20s %2d) %-20s\r\n", y+1, armsmods[y], y+2, y+1 < NUM_ARMSMODS ? armsmods[y+1] : "");
+  sprintbit(GET_OBJ_VAL(OBJ, 3), armsmods, buf1, sizeof(buf1));
```

Comment:
This is going to add more magic numbers to the code, with magic numbers being defined as 'numbers that aren't explained, but have an effect on the code if changed'. Recommend swapping out the `GET_OBJ_VAL(OBJ, 3)` calls with something from utils.h like `GET_CYBERWARE_FLAGS(OBJ)`.